src/main/groovy/de/gesellix/gradle/docker/models/DockerContainer.groovy

```diff
@@ -0,0 +1,393 @@
+package de.gesellix.gradle.docker.models
```

Comment:
ok, now I get the idea. Though I'm still thinking about a better package and class name, I don't have _the perfect idea_ ;-) yet, so let's keep it for now.